site/en/docs/privacy-sandbox/chips/index.md

```diff
@@ -6,7 +6,7 @@ subhead: >
 description: >
   CHIPS is a Privacy Sandbox proposal that will allow cookies to be set by a third-party service, but only read within the context of the top-level site where they were initially set
```

Comment:
While you're in here, could you duplicate this into the `subhead` field too so that it gets a little preview text on the https://deploy-preview-2474--developer-chrome-com.netlify.app/docs/privacy-sandbox/ page ?